### ISSUE TITLE
支持在WebGL加载AssetBundle吗

### ISSUE THREAD

**Issue by clksaaa** [tuyoogame/YooAsset#10]
支持在WebGL加载AssetBundle吗

**Comment by gmhevinci** [comment]
目前只支持offlinePlayMode

**Comment by clksaaa** [comment]
是指将资源拷贝到StreamingAssets 然后使用离线模式是吗




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年5月12日(星期四) 晚上8:51
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [tuyoogame/YooAsset] 支持在WebGL加载AssetBundle吗 (Issue #10)





 
目前只支持offlinePlayMode

**Comment by gmhevinci** [comment]
是的~ 目前支持三种运行模式：EditorSimulateMode, OfflinePlayMode, HostPlayMode，以及后续完全支持WebGL的新模式，WebPlayMode.

**Comment by AILHC** [comment]
现在的资源包搜集工具的undo有点尴尬。group内的增加删除收集配置不支持undo/redo，删除了，没法undo和redo

**Comment by clksaaa** [comment]
> 
那真是太好了 赞

**Comment by HiWenHao** [comment]
我来问问，2021版本的WebGL支持了么。

**Comment by gmhevinci** [comment]
已经支持WebGLl

**Comment by HiWenHao** [comment]
好的大哥，我已经用到了


?王闻浩
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tuyoogame/YooAsset"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月3日(星期四) 中午12:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [tuyoogame/YooAsset] 支持在WebGL加载AssetBundle吗 (Issue #10)





 
已经支持WebGL

**Comment by csy7749** [comment]
![Uploading {V46S`JL@[Q@}Z([JE8H`]O.png…]()
我使用web下的offline模式加载示例项目的这个bundle就会报错

**Comment by csy7749** [comment]
![ZHKMW4G_JOE{1BRLIHVUJ4Y](https://user-images.githubusercontent.com/44502734/222351441-7bc9abfe-94aa-43bd-bcef-8e609f1cb0c9.png)

**Comment by HiWenHao** [comment]
> ![ZHKMW4G_JOE{1BRLIHVUJ4Y](https://user-images.githubusercontent.com/44502734/222351441-7bc9abfe-94aa-43bd-bcef-8e609f1cb0c9.png)

你得先把错误完整的放出来，通过目前错误我无法看出错误所在，文西哥在忙，有空会及时回复✍

**Comment by csy7749** [comment]
![image](https://user-images.githubusercontent.com/44502734/222361582-0a5f7859-6606-4094-88e1-accde018aeae.png)
![image](https://user-images.githubusercontent.com/44502734/222361625-adcd9cf6-c56a-40c6-9072-c408e93432d0.png)
这就是完整的了，我测试了Android版和pc版没得问题。

**Comment by HiWenHao** [comment]
那你等等文西哥吧，我也好久没看了。说不定现在有专门针对WebGL版本，或者说案例已经很久没更新了，不如去QQ群里问他。

**Comment by csy7749** [comment]
qq问了，还没的大佬回我T.T

**Comment by gmhevinci** [comment]
尝试在打包界面，把加密选项设置为None.
